mlfinlab/tests/test_fingerpint.py

```diff
@@ -0,0 +1,125 @@
+"""
+Test RegressionModelFingerprint, ClassificationModelFingerprint implementations
+"""
+
+import unittest
+import pandas as pd
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.linear_model import LinearRegression
+from sklearn.datasets import load_boston
+from mlfinlab.feature_importance import RegressionModelFingerprint
+
+
+# pylint: disable=invalid-name
+
+class TestModelFingerprint(unittest.TestCase):
+    """
+    Test model fingerprint functions
+    """
+
+    def setUp(self):
+        """
+        Set the file path for the sample dollar bars data.
+        """
+
+        self.X, self.y = load_boston(return_X_y=True)
+        self.X = pd.DataFrame(self.X[:100])
+        self.y = pd.Series(self.y[:100])
+
+        self.reg_1 = RandomForestRegressor(n_estimators=10, random_state=42)
+        self.reg_2 = LinearRegression(fit_intercept=True, normalize=False)
+        self.reg_1.fit(self.X, self.y)
+        self.reg_2.fit(self.X, self.y)
+
+        self.reg_1_fingerprint = RegressionModelFingerprint(self.reg_1, self.X, num_values=20)
+        self.reg_2_fingerprint = RegressionModelFingerprint(self.reg_2, self.X, num_values=20)
+
+        self.reg_1_fingerprint.fit()
+        self.reg_2_fingerprint.fit()
+
+    def test_linear_effect(self):
```

Comment:
We should add the test that the gradient of the linear model outputs should be the linear reg coef for that feature, if a set is created with make_regression from sklearn. 

The main thing is to add a unit test from the paper that says the slope of linear model on $x_k$ should be the coef. I will elaborate over a phone call.